lib/srv/db/proxyserver.go

```diff
@@ -660,7 +668,7 @@ func getConfigForClient(conf *tls.Config, ap auth.ReadDatabaseAccessPoint, log l
 				log.Debugf("Ignoring unsupported cluster name %q.", info.ServerName)
 			}
 		}
-		pool, err := auth.ClientCertPool(ap, clusterName)
+		pool, err := auth.ClientCertPool(ap, clusterName, types.HostCA, types.DatabaseCA, types.UserCA)
```

Comment:
Do we need to load all CAs here? Client certificates are signed by user CA, no?

Reply:
When I refactored the function I wanted to keep the same sets of certs as before, but I agree. We need only `UserCA`. Changed